cmd/ketch/cluster_issuer.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+)
+
+var (
```

Comment:
could we avoid global vars? how about `getclusterIssuerGVR()` ?

Reply:
what is the problem with this type of usage?

Reply:
remove the var declaration